packages/pangea-sdk/src/test/java/cloud/pangeacyber/pangea/audit/ITAuditTest.java

```diff
@@ -1140,4 +1142,41 @@ public void testLogStream() throws ConfigException, PangeaAPIException, PangeaEx
 		final var response = client.logStream(input);
 		assertEquals(ResponseStatus.SUCCESS.toString(), response.getStatus());
 	}
+
+	@Test
+	public void testExportDownload() throws PangeaAPIException, PangeaException {
+		var exportResponse = clientGeneral.export(ExportRequest.builder().start("1d").verbose(true).build());
+		assertEquals(ResponseStatus.ACCEPTED.toString(), exportResponse.getStatus());
+
+		// Note that the export can easily take dozens of minutes, if not
```

Comment:
can we have one of the SDKs do one test run somewhere (maybe once every 50 test runs or something on main) that tries to poll for the full response?

Reply:
Test now polls for the last 1 day of results, like the other SDKs.